### DISCUSSION TITLE
Don't see github copilot in Windows 11 parallel desktop in Macbook Pro M1 MAX

### DISCUSSION THREAD

**Discussion by emranashraf** [community/community#43779] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

I'm trying to install GitHub Copilot on Windows 11 Visual Studio 2022 17.4.3 installed in Parallel Desktop on my Macbook Pro M1 MAX. When I try to search "Copilot" extension in Visual Studio it's not showing anything. 

Here are details

Hardware: Macbook Pro M1 MAX
OS: Windows 11 installed on Parallel Desktop
Visual Studio 2022 for windows version 17.4.3

![Screenshot 2023-01-09 at 5 15 30 PM](https://user-images.githubusercontent.com/5232556/211419672-dc960868-4b73-4f5c-a7fc-9cede1c81b89.png)

![Screenshot 2023-01-09 at 5 19 48 PM](https://user-images.githubusercontent.com/5232556/211419891-e5419d38-8e09-4b8e-ada9-a11e5e260b2a.png)

**Comment by iamcjwilliams** [comment]
Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Reply by emranashraf** [reply]
Hi @iamcjwilliams
Thanks for your response, but the URL you have provided is not working.
Thanks

**Reply by iamcjwilliams** [reply]
@emranashraf Provided updated link

**Reply by emranashraf** [reply]
Hi @iamcjwilliams, The URL you have provided is to troubleshoot issues after installation of copilot but in my case I can't event install copilot. When I search copilot extension, nothing is showing up.

**Reply by tilmann1** [reply]
@emranashraf The Copilot VS extension currently has no support for the Arm64 architecture, e.g. Apple Silicon hardware. The only supported platform is x86_64 at the moment. This is the reason why you can't see the extension when you search for it in the Visual Studio Marketplace.

**Reply by emranashraf** [reply]
Thanks @tilmann1

**Reply by LukeStephen** [reply]
Any plans to support windows on ARM now that visual studio supports it? Any alpha bits I can help test?

**Reply by lunasoft2001** [reply]
It's that not a Solution? 
https://devblogs.microsoft.com/visualstudio/now-introducing-arm64-support-for-vs-extensions/

**Reply by Wowotoe** [reply]
Copilot just updated their VS requirements to 17.4+ last week. But VS Installer for 17.4 cannot selects AMD version under Win11 ARM. I think there are many developers out there are using Mac/Parallels/VS for development. To get it to work, you need to install VS 2022 17.2 with older version of Copilot extension, if you can find it.

**Reply by grovolis** [reply]
Any way to do this without downgrading yet?

**Reply by tilmann1** [reply]
I can confirm that https://github.com/community/community/discussions/43779#discussioncomment-4680735 is still true. GitHub continues to monitor customer feedback for Arm64 demand to factor into future product decisions. Thanks!

**Reply by fobrs** [reply]
Please add an arm64 capable version asap!

**Reply by malaksedarous** [reply]
If the future is ARM and if Microsoft is trying to push ARM development, then Copilot for ARM is very important.

**Comment by LightspeedNZ** [comment]
Hi I'd like to see this as well!

**Reply by fobrs** [reply]
You can install it by hand. I did it and it works! Unzip and copy contents to Extensions folder.

**Reply by HerbWarren** [reply]
@fobrs Where did you get the extension to unzip? Looks like I'd need an Intel install of VS2022 to d/l the extension...

**Reply by fobrs** [reply]
> @fobrs Where did you get the extension to unzip? Looks like I'd need an Intel install of VS2022 to d/l the extension...

https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Reply by brucegomes** [reply]
@fobrs  that marketplace link you provided says it's for amd64. will it still work on windows arm inside parallels ?

**Reply by fobrs** [reply]
unzip it, and install it by hand....

**Comment by lunasoft2001** [comment]
You can find the extension NOW direct in VS 2022 (arm version). But I think it's a preview version

**Reply by grovolis** [reply]
> You can find the extension NOW direct in VS 2022 (arm version). But I think it's a preview version

You can indeed, I was able to install it and seems to work fine. I don't think it's the proper Copilot though, it's the new AI chat feature I think?

**Comment by ChrisHolcomb** [comment]
So Microsoft has Copilot for VS Code on a Mac with M2 chip, but doesn't release a version for Visual Studio 2022 running on windows 11 arm?

**Comment by johnfager** [comment]
I'd like to see this as well. A lot of us are using the Mac M2 chips and are .net developers with Parallels. I would think that cross platform has been a big push for Microsoft. This feels like an important tool for developers, so it's very disappointing that it is not released to work on ARM64.

**Comment by malaksedarous** [comment]
Help by upvoting this https://developercommunity.visualstudio.com/t/Can-we-have-Github-Copilot-support-for-V/10325883

**Comment by skay97** [comment]
Since the extension still hasn't been officially ported by Microsoft, you can do it manually like so: 
https://stackoverflow.com/a/76737023
Copilot now works on my arm installation of visual studio in parallels.

**Reply by Wowotoe** [reply]
Thanks skay97. The 2nd solution from Stackoverflow link (replacing the contents in Extension folder) works for me. Now I have both Copilot and Chat available in VS 2022 v17.8.0 ARM Preview
